pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -1249,6 +1249,9 @@ func TestOutOfDiskNodeDaemonLaunchesCriticalPod(t *testing.T) {
 		utilfeature.DefaultFeatureGate.Set("ExperimentalCriticalPodAnnotation=True")
 		syncAndValidateDaemonSets(t, manager, ds, podControl, 1, 0, 0)
 	}
+
+	// Rollback feature gate to default value.
+	utilfeature.DefaultFeatureGate.Set("ExperimentalCriticalPodAnnotation=False")
```

Comment:
To rollback you need to check what the original value is (use `utilfeature.DefaultFeatureGate.Enabled` to get it).